README.md

```diff
@@ -82,10 +82,10 @@ We support evaluation datasets for `Text -> Text`, `Text + Image -> Text` and `T
 

```

Comment:
What about using `under construction` or `coming soon` other than `in the planning`